docs/_includes/top_nav_home.html

```diff
@@ -1,44 +1,34 @@
 <nav class="navbar home navbar-expand-lg">
```

Comment:
did we want to remove the 'beta' at the top of the page?

Reply:
nope! good catch